.github/actions/fetch-releases-json/action.yaml

```
@@ -35,7 +35,7 @@ runs:
         if [[ "${{ inputs.image-name }}" = "*" ]]; then
           echo "Copying all _releases.json files"
           cd ${{ inputs.directory }}
-          find . -name "_releases.json" -exec cp --parents {} $OLDPWD/
+          find . -name "_releases.json" -exec cp --parents {} $OLDPWD/ \;
```

I believe you have an extra `/`

```suggestion
          find . -name "_releases.json" -exec cp --parents {} $OLDPWD \;
```